src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/BindingMetadataProviderContext.cs

```diff
@@ -36,6 +39,16 @@ public class BindingMetadataProviderContext
         public ModelMetadataIdentity Key { get; }
 
         /// <summary>
+        /// Gets the property attributes.
+        /// </summary>
+        public IReadOnlyList<object> PropertyAttributes { get; }
```

Comment:
That would award a lot of #PranavPoints! I can't believe I didn't think of this.